hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -290,16 +288,42 @@ By default, no extra debug messages are enabled.
 XHCI_DEBUG=0x3
 ```
 Default: 0x0  
-Version: pieeprom-2020-06-15.bin - STABLE  
+Version: pieeprom-2020-09-03.bin  
 
 ## config.txt - configuration properties
-### BOOT_LOAD_FLAGS
+### boot_load_flags
 Experimental property for custom firmware (bare metal).
 
 Bit 0 (0x1) indicates that the .elf file is custom firmware. This disables any compatiblity checks (e.g. is USB MSD boot supported) and resets PCIe before starting the executable. 
 
 Default: 0x0  
-Version: pieeprom-2020-06-15.bin - STABLE  
+Version: pieeprom-2020-09-03.bin 
+
+### uart_2ndstage
+If set to 0x1 then enable debug logging to the UART. In newer firmware versions (Raspberry Pi OS 2020-08-20 and later) UART logging is also automatically enabled in start.elf. This is also described on the [Boot options](../..//configuration/config-txt/boot.md) page.
+
+Default: 0x0  
+Version: pieeprom-2020-09-03.bin 
+
+### eeprom_write_protect  
+Controls whether the bootloader and VLI EEPROMs are marked as write protected.
+
+** This has no effect unless the EEPROM /WP pin is pulled low (TP5). Similarly, /WP low only write protects the status register so if EEPROM write protect was not previously defined then the EEPROM would not actually be write protected **
+
+See: [Winbond W25x40cl datasheet](https://www.winbond.com/resource-files/w25x40cl_f%2020140325.pdf)
+
+* 1 - Confiures the write protect regions to cover the entire EEPROM.
+* 0 - Clears write protect regions
+* -1 - Do nothing.
+
+Default: -1  
+Version: pieeprom-2020-09-03.bin 
+
+### bootloader_update
+This option may be used to block self-update without requiring the EEPROM configuration to be updated. This is sometimes useful when updating multiple Pis via network boot because this option can be controlled per Raspberry Pi (e.g. via a serial number filter in config.txt).
```

Comment:
Perhaps change "This option may be used to block..." to "This option may be set to 0 to block..." ?

Reply:
done